packages/vertexai/src/methods/chrome-adapter.test.ts

```diff
@@ -298,4 +298,87 @@ describe('ChromeAdapter', () => {
       });
     });
   });
+  describe('countTokens', () => {
+    it('With no initial prompts', async () => {
```

Comment:
Minor: describing this as a test for a singular input, eg `it('counts tokens from a singular input'`, would be more self-explanatory from my perspective.